cpp/src/aztec3/circuits/abis/barretenberg/verifier_reference_string.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
```

Comment:
Same for this. Can we move this to bb?

Reply:
This one is a bit hacky and just needed for circuits.js - does upstream BB want it? If that doesn't pose a problem to BB then I can move it